frontend/style.css

```diff
@@ -0,0 +1,91 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+html {
+	height: 100%;
+}
+article, aside, details, figcaption, figure, footer, header, hgroup, main, nav, section, summary {
+	display: block;
+}
+body {
+	font: Montserrat,sans-serif;
+    font-size: 15px;
+	height: 100%;
+}
+.wrapper {
+	width: 100%;
+	min-width: 1000px;
+	min-height: 100%;
+	height: auto !important;
```

Comment:
try to avoid the "important" directive, it is very difficult to work with it later